dwhdelft.nl/pages/mixup.vue

```diff
@@ -0,0 +1,221 @@
+            <ElementsParagraphedText :paragraphs="t('intro')" class="md:text-xl md:leading-relaxed space-y-4" />
+            <div class="flex flex-1 space-x-16">
+              <div>
+                <div class="flex flex-1 space-x-8 space-y-16 md:space-y-8 lg:space-y-6">
+                  <div>
+                    <p
+                      v-if="barOpeningHours.announcement"
+                      class="mt-3 mb-4 text-brand-500"
+                      v-text="tt(barOpeningHours.announcement)"
+                    />
+                    <PagesHomeInvite
+                      class="my-8 md:my-4"
+                      :announcement="t('invite.announcement')"
+                      :time="t('invite.time', [barOpeningHours.start_time])"
+                    />
+                  </div>
+                  <div>
+                    <MIXUPLogo class="h-20" />
+                  </div>
+                </div>
+                <div class="flex flex-1 space-x-4">
+                  <div>
+                    <a href="https://my.dwhdelft.nl/signup">
+                      <ElementsSecondaryButton class="!text-brand-450" arrow>
+                        {{ t('membership_button') }}
+                      </ElementsSecondaryButton>
+                    </a>
+                  </div>
+                  <div>
+                    <nuxt-link :to="localePath('barbuddy')">
+                      <ElementsPrimaryButton class="!text-brand-50" arrow>
+                        {{ t('barbuddy_button') }}
+                      </ElementsPrimaryButton>
+                    </nuxt-link>
+                  </div>
+                </div>
+              </div>
+              <div class="hidden 2xl:block">
+                <div>
+                  <img src="~/assets/images/photos/mixup/barnight.jpg" class="w-64" />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </ElementsContainer>
+    </section>
+
+    <section class="bg-highlights bg-cover bg-center py-10">
+      <ElementsContainer>
+        <div class="grid md:grid-cols-2 lg:grid-cols-4 gap-3">
+          <ElementsActionCard v-for="point in t('bulletPoints')" :key="point.title" :title="point.title">
+            <template #header>
+              <div class="w-full h-40 overflow-hidden mx-auto">
+                <img :src="imageOverviews(point.image)" class="object-cover w-full h-full" />
+              </div>
+            </template>
+            <p class="-mt-3 text-brand-900 text-lg leading-snug" v-text="point.description" />
+          </ElementsActionCard>
+        </div>
+      </ElementsContainer>
+    </section>
+
+    <section class="bg-brand-900 pb-12">
+      <ElementsContainer>
+        <div class="mx-auto pt-12 pb-8">
+          <h1 class="text-center text-white font-medium text-5xl mb-6 leading-tight" v-html="t('highlights.title')" />
+        </div>
+        <div class="grid grid-cols-[1fr_minmax(0,1280px)_1fr]">
```

Comment:
ok om dit voor nu zo te laten, maar in de toekomst gaan we dit ronde componentje nog even moeten aanpakken, zeker omdat er vierkante foto's instaan met tekst die wordt afgekapt atm